src/CodeFormatter/Program.cs

```diff
@@ -117,11 +131,42 @@ private static int RunFormatCommand(FormatOptions options)
 
                 return FAILED;
             }
+            finally
+            {
+                stopwatch.Stop();
+                Console.WriteLine("Total time: {0}", stopwatch.Elapsed);
+            }
         }
 
-        private static async Task<int> RunFormatAsync(FormatOptions options, CancellationToken cancellationToken)
+        private static IAnalyzerAssemblyLoader _analyzerAssemblyLoader;
+        private static IAnalyzerAssemblyLoader AnalyzerAssemblyLoader {
+            get
+            {
+                if(_analyzerAssemblyLoader == null)
+                {
+                    // use Roslyn's existing simple loader, there're no special requirements for our usage
+                    var loaderAssembly = Assembly.Load((typeof(CommandLineProject)).Assembly.FullName);
+                    _analyzerAssemblyLoader = (IAnalyzerAssemblyLoader)Activator.CreateInstance(loaderAssembly.GetType("Microsoft.CodeAnalysis.SimpleAnalyzerAssemblyLoader"));
```

Comment:
Good question, I'll investigate. I originally started to write my own trivial one and then saw the Roslyn version and figured 'SimpleAnalyzerAssemblyLoader' sounded like just what I needed :)